### PR TITLE
Add Katello 3.17 release docs

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

**Comment by jturel** [comment]
API docs are missing & there aren't any release notes - feel free to reach out to the team to help put those together but the changelog should be a good guide to pluck the highlights!

edit which reminds me - speaking of API docs, I think we need a hammer-cli-katello-0.23 release?

File: _data/plugins/katello.yml [hunk 1]
```diff
@@ -1,4 +1,6 @@
 versions:
+  - name: "3.17"
+    display: "3.17 RC 1"
```
Comment (jturel) [inline _data/plugins/katello.yml]: nit: "RC2" (no space)

**Comment by chris1984** [comment]
@jturel updated

**Review by jturel — APPROVED** [review]
ACK!

**Review by tbrisker — COMMENTED** [review]
not sure why, but the apidocs seem to have both `en` and `en_US` versions, we can probably drop one of them.

**Comment by chris1984** [comment]
@tbrisker 3.16 and 3.16 have it too, so I copied it to match. 

https://github.com/theforeman/theforeman.org/tree/gh-pages/plugins/katello/3.16/api/apidoc

https://github.com/theforeman/theforeman.org/tree/gh-pages/plugins/katello/3.15/api/apidoc

**Comment by chris1984** [comment]
@jturel updated with removing the `en_US` files and api docs still render correctly.

**Review by jturel — CHANGES_REQUESTED** [review]
Still seeing a lot of JSON which makes me think you missed this step:

https://github.com/theforeman/tool_belt/blob/8569eff9750ad953247b3ffb34c6d0bf01ff746e/procedures/katello/branch.md.erb#L55-L56

**Comment by chris1984** [comment]
@jturel yeah that was my bad, updated to remove the .json files

**Comment by jturel** [comment]
Looks like all feedback is addressed. Merging so we can get the announcement out!

**Pull request merged**